packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -618,14 +618,20 @@ When the `selectionBehavior` prop is set to `"replace"`, clicking a row with the
 ```
 
 ### Row and cell actions
+`useTable` supports row actions via the `onRowAction` prop and cell actions via the `onCellAction` prop, both of which is useful for functionality such as navigating into items to open them or get more details.
```

Comment:
Open question: should I get rid of the cell actions in the example and description? It seems to complicate the behavior quite a bit and is a weird experience when combined with row actions.

Reply:
Got rid of them for now since it feels buggy and we haven't tested it. Will make followup ticket